tests/models/cpmant/test_tokenization_cpmant.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# Copyright 2022 The OpenBMB Team and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from transformers.testing_utils import custom_tokenizers, is_torch_available
+
+from ...test_tokenization_common import TokenizerTesterMixin
+
+
+if is_torch_available():
+    from transformers.models.cpmant.tokenization_cpmant import CPMAntTokenizer
+
+
+@custom_tokenizers
+class CPMAntTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
+    def test_pre_tokenization(self):
```

Comment:
can you add the same test for `CPMAntTokenizerFast` ?

Reply:
Still needs to be addressed.